curator/api/utils.py

```diff
@@ -310,6 +310,7 @@ def prune_allocated(client, indices, key, value):
     :arg indices: A list of indices to act on
     :arg key: The allocation attribute to check for
     :arg value: The value to check for
+    :arg allocation_type: type of allocation
```

Comment:
Please try to follow the descriptive patterns and capitalization.  This could be something like "The allocation type to check for.  Must be one of 'require', 'include', or 'exclude'."